### DISCUSSION TITLE
如果我把原神5.0直播弹幕内容数据可视化？

### DISCUSSION THREAD

**Discussion by Hutaodegou1** [share121/inter-knot#1520] [其他]
![006cgq6Hly1hfogfatv14j306o06o0sx](https://github.com/user-attachments/assets/234cf193-b09f-4ad7-ae1a-d4dfa60c9c05)

下面是我之前得到的b站评论结果。

现在我想做个程序，搞一下原神5.0直播的弹幕可视化，并且结果实时更新，同步直播。

艾梅莉埃 PV 共24796条
单个uid评论数大于5个的，占总有效评论的百分比  28%
再下面是无效总数 6960

忘忧谐律之歌 155160条
单个uid评论数大于5个的，占总有效评论的百分比  67%
再下面是无效总数 104992

艾梅莉埃 角色演示 51892条
单个uid评论数大于5个的，占总有效评论的百分比 38%
再下面是无效总数 19804

**Comment by ShenYouJiaA** [comment]
支持

**Comment by jiaopanda** [comment]
加油，支持你

**Comment by OoTyleroO** [comment]
支持！

**Comment by Loxia1121** [comment]
加油！

**Comment by fu-fu404** [comment]
支持，5.0已经有人爆出要上演智械危机了

**Comment by capacitor1** [comment]
支持

我只能得到所有的弹幕，却没办法实时处理。。就看你了

**Comment by tyhpoi** [comment]
支持，您就是传说中的技术宅吗？

**Comment by Elysia-Eden-11** [comment]
支持，嘿嘿

**Comment by AnbulesPeo** [comment]
支持你朋友，虽然我只想看结论。。。

**Comment by Ste11er** [comment]
支持

**Comment by xintie2023** [comment]
支持

**Comment by szlx2333** [comment]
支持

**Comment by jis1234** [comment]
这个委托我接了

**Comment by Tabellarii** [comment]
支持(/•ิ_•ิ)/

**Comment by 11-billy** [comment]
支持
